ui/src/app/views/dataset/component.tsx

```diff
@@ -0,0 +1,147 @@
+        .getConceptSetsInWorkspace(namespace, id));
+      this.setState({conceptSetList: conceptSets.items});
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  render() {
+    const {namespace, id} = this.props.workspace;
+    const {creatingConceptSet, conceptDomainList, conceptSetList} = this.state;
+    return <React.Fragment>
+      <FadeBox style={{marginTop: '1rem'}}>
+        <h2 style={{marginTop: 0}}>Datasets</h2>
+        <div style={{color: '#000000', fontSize: '14px'}}>Build a dataset by selecting the
+          variables and values for one or more of your cohorts. Then export the completed dataset
+          to Notebooks where you can perform your analysis</div>
+        <div style={{display: 'flex'}}>
+          <div style={{marginLeft: '1.5rem', marginRight: '1.5rem', width: '33%'}}>
+            <h2>Select Cohorts</h2>
+            <div style={{backgroundColor: 'white', border: '1px solid #E5E5E5'}}>
+              <div style={styles.selectBoxHeader}>
+                Cohorts
+                <ClrIcon shape='plus-circle' class='is-solid' style={styles.addIcon}
+                  onClick={() => navigate(['workspaces', namespace, id,  'cohorts', 'build'])}/>
+              </div>
+              {/*TODO: load cohorts and display here*/}
+              <div style={{height: '8rem'}}/>
+            </div>
+          </div>
+          <div style={{width: '58%'}}>
+            <h2>Select Concept Sets</h2>
+            <div style={{display: 'flex', backgroundColor: 'white', border: '1px solid #E5E5E5'}}>
+              <div style={{flexGrow: 1}}>
+                <div style={{...styles.selectBoxHeader, borderRight: '1px solid #E5E5E5'}}>
+                  Concept Sets
+                  <ClrIcon shape='plus-circle' class='is-solid' style={styles.addIcon}
+                           onClick={() => this.setState({creatingConceptSet: true})}/>
+                </div>
+                {/*TODO: display concept sets here*/}
+                <div style={{height: '8rem', borderRight: '1px solid #E5E5E5'}}/>
+              </div>
+              <div style={{flexGrow: 1}}>
+                <div style={styles.selectBoxHeader}>
+                  Values
+                </div>
+                {/*TODO: load values and display here*/}
+                <div style={{height: '8rem'}}/>
+              </div>
+            </div>
+          </div>
+        </div>
+      </FadeBox>
+      <FadeBox style={{marginTop: '1rem'}}>
+        <div style={{backgroundColor: 'white', border: '1px solid #E5E5E5'}}>
+          <div style={{...styles.selectBoxHeader, display: 'flex', position: 'relative'}}>
+            <div>Preview Dataset</div>
+            <div style={{marginLeft: '1rem', color: '#000000', fontSize: '14px'}}>A visualization
+              of your data table based on the variable and value you selected above</div>
+            <Button style={{position: 'absolute', right: '1rem', top: '.25rem'}} disabled={true}>
```

Comment:
why setting disabled to true here?  if it's for code that hasn't been written yet add a comment?